src/Codeception/Module/DataFactory.php

```diff
@@ -173,6 +170,16 @@ public function _beforeSuite($settings = [])
             }
         }
     }
+    
+    /**
+     * @return StoreInterface
+     */
+    protected function getStore()
+    {
+        return $this->ormModule instanceof DataMappe
```

Comment:
Yeah, this module really lacks integration tests :(
To be honest it is pretty hard to set up them.